services/userService.js

```diff
@@ -0,0 +1,61 @@
+const User = require("../models/user");
+
+async function createUser(data) {
+  try {
+    const user = await User.create(data);
+    return user;
```

Comment:
The return statement is not wrapped in a consistent response format. Consider using a standardized response format throughout the service, such as an object with a 'data' property, to improve code readability and maintainability.